.env.docker.dev

```diff
@@ -3,6 +3,7 @@ FREQUENCY_URL=ws://frequency:9944
 REDIS_URL=redis://redis:6379
 PROVIDER_ID=1
 PROVIDER_BASE_URL=http://webhook:3001/api/v1.0.0
+CONNECTIONS_PER_PAGE=100
```

Comment:
nit: suggest `CONNECTIONS_PER_PROVIDER_RESPONSE_PAGE` or something; let's be clear that it's related to the Provider response and not the on-chain graph page